From 75303b1ea35ffaffcdfedc43e08d10b572559722 Mon Sep 17 00:00:00 2001
From: Tomasz Lauda <tomasz.lauda@linux.intel.com>
Date: Thu, 15 Mar 2018 17:31:59 +0100
Subject: [PATCH] cnl: don't switch to lower clock

We don't want clock optimizations right now, so stay with 400 MHz clock.

Signed-off-by: Tomasz Lauda <tomasz.lauda@linux.intel.com>
---
 src/platform/cannonlake/platform.c | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/src/platform/cannonlake/platform.c b/src/platform/cannonlake/platform.c
index cf7cce8..476659e 100644
--- a/src/platform/cannonlake/platform.c
+++ b/src/platform/cannonlake/platform.c
@@ -144,9 +144,6 @@ int platform_boot_complete(uint32_t boot_message)
 	mailbox_dspbox_write(sizeof(ready), &sram_window,
 		sram_window.ext_hdr.hdr.size);
 
-	/* boot now complete so we can relax the CPU */
-	clock_set_freq(CLK_CPU, CLK_DEFAULT_CPU_HZ);
-
 	/* tell host we are ready */
 	ipc_write(IPC_DIPCIDD, SRAM_WINDOW_HOST_OFFSET(0) >> 12);
 	ipc_write(IPC_DIPCIDR, 0x80000000 | SOF_IPC_FW_READY);
-- 
2.7.4

